cloud-controller-manager/do/loadbalancers.go

```diff
@@ -218,6 +220,10 @@ func (l *loadBalancers) EnsureLoadBalancer(ctx context.Context, clusterName stri
 	}
 
 	if lb.Status != lbStatusActive {
+		if lb.Status == lbStatusNew {
+			return nil, api.NewRetryError("load-balancer is currently being created", 15*time.Second)
+		}
+
 		return nil, fmt.Errorf("load-balancer is not yet active (current status: %s)", lb.Status)
```

Comment:
I'd rephrase this now that any status other than "active" is unexpected and treated as a genuine error:

```suggestion
		return nil, fmt.Errorf("load-balancer has unexpected status %q", lb.Status)
```

Reply:
Addressed.